### PR TITLE
Updated get-testnet-tokens.md and account-creation.md

### PR DESCRIPTION
**Pull request by afmsavage**
I updated the contents of the get-testnet tokens tutorial and some of the account-creation tutorial since it is referenced in the get-testnet tokens tutorial.  I was unable to get any answers from Telegram or the forums on if the edgenet was still alive, but I was unable to connect to any of the URLs or endpoints of it so I removed it from the tutorials.  I can readd it if it is still alive.  Please let me know if you would like me to make any additional changes to these tutorials.  I will begin my work on the others :)

AE Address: ak_Fowfckktub4tsPW35mfUwYSjhDxCLdEYacNeCx2qMhtSVXFLb

### CODE REVIEW

**Comment by thepiwo** [comment]
some of the tutorials were already updated, can you make separate PRs for any tutorial that has not been updated yet?

**Comment by thepiwo** [comment]
I'll close this, feel free to reopen if only non-update tutorials are adjusted

File: get-testnet-tokens.md [hunk 1]
```diff
@@ -1,63 +1,79 @@
...
 ## Prerequisites
+
 - Please read our previous tutorial [How to Create an æternity Account With CLI?](account-creation-in-ae-cli.md) in order to better understand the next steps and to follow them.
+
 ## æternity test networks
-Testnets are copies of the æternity blockchain almost identical to the live network except the fact that their AE tokens are worthless.
+
+Testnets are copies of the æternity blockchain almost identical to the live network except for the fact that their AE tokens are worthless.
 
 The æternity SDK team currently runs three testnets - **sdk-testnet**, **sdk-edgenet** and **uat-testnet**. Testnet is the latest stable release, and Edgenet is the next. The æternity core team runs another network, **uat-testnet**. It’s for internal testing and is not guaranteed to work with the SDKs. But it’s the one which has the very latest features and may be useful for you if you are an enthusiast who runs their own nodes.
```
[the remark this answers is not in the document]
Comment (afmsavage) [reply get-testnet-tokens.md]: I'm not sure.  I asked on the forums and did not get a reply.  I could not find any information about the  sdk-edgenet and well uat-testnet is personally run networks so I left that.  I can change this if required.

File: account-creation-in-ae-cli.md [hunk 2]
```diff
@@ -46,59 +61,83 @@ Commands:
...
 ```
-aecli account create <name> [options] 
+aecli account create <name> [options]
 ```
+
 - <name> - the name of the wallet
 - Тhe option capabilities of the command are the following:
+
 ```
+Usage: aecli-account [options] [command]
```
[the remark this answers is not in the document]
Comment (afmsavage) [reply account-creation-in-ae-cli.md]: changing this in a new update to this pull request.

**Comment by afmsavage** [comment]
@thepiwo so I did all this work for nothing?  If there were PRs already submitted before mine 15 days ago, why weren't they merged prior?  Was mine included?  Spending time on these when they didn't need to be updated.  Feel cheated.

**Comment by thepiwo** [comment]
Sorry I don't know how to answer this, I tried to merge in the order they did come in, once they were finalized and reviewed. Also I tried not to merge similar changes on the same tutorial, if it worked fine prior.

Please don't feel cheated, if you find this behavior unfair please contact the organizers of the bounty program to find an agreement. I do just overlook the content of this repository and am not involved with the bounty program.

**Pull request closed without merging**